runtime/src/bank.rs

```diff
@@ -8623,8 +8623,8 @@ pub(crate) mod tests {
         let zero_lamports = 0;
         let little_lamports = 1234;
         let large_lamports = 123_456_789;
-        let some_slot = 1000;
-        let rent_collected = 22; // this is a function of 'some_slot'
+        let some_slot = 32; // chosen to cause epoch to be +1
```

Comment:
Would it be possible to use fields within `genesis_config.epoch_schedule` to compute this value?

Reply:
used a constant